src/style.css

```diff
@@ -195,3 +203,25 @@ h1 {
 .left-div-li {
   width: 100%;
 }
+
+.red-badge {
+  font-size: smaller;
+}
+
+.badge2 {
+  position: absolute;
+  top: 3px;
+  right: 3px;
+  display: inline-block;
+  min-width: 10px;
+  padding: 2px 3px;
+  font-size: 9px;
+  font-weight: 400;
+  line-height: 1;
+  color: #fff;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: baseline;
+  background-color: red;
+  border-radius: 10px;
+}
```

Comment:
- [ ] Good job so far your project is impressive. It works as expected :100: But I would like to recommend you reduce the `width` of your todo list website displays a list that looks and behaves like the part of the minimalist project captured in the video below :arrow_down: 

#### Capture design
![68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067 (1)](https://user-images.githubusercontent.com/6615394/199602116-1bdf17f9-d775-4812-af03-24d2fc870893.jpeg)

#### Your design
![Screenshot from 2022-11-02 20-55-41](https://user-images.githubusercontent.com/6615394/199602142-1ee7cded-d717-4cc9-8f0b-a0cc65981fbe.png)

Reply:
Hi! @codecaiine i set the max-with in 490 px as the real site in https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/.
Regards!

![image](https://user-images.githubusercontent.com/53025062/199603578-8421d295-ea01-45b4-9ad4-11f71ca6dee7.png)